golang/cosmos/daemon/cmd/root.go

```diff
@@ -263,6 +283,58 @@ func (ac appCreator) newApp(
 	)
 }
 
+const (
+	FlagExportDir                   = "export-dir"
+	ExportedGenesisFileName         = "genesis.json"
+	ExportedSwingStoreDirectoryName = "swing-store"
+)
+
+func extendCosmosExportCommand(cmd *cobra.Command, hasVMController bool) {
+	cmd.Flags().String(FlagExportDir, "", "The directory where to create the genesis export")
```

Comment:
I didn't add a help string for `FlagSwingStoreExportDir` as from what I understand it could technically be set by config as well, not just by the command line. I have to admit I'm a little fuzzy with how the whole config thing is plumbed in cosmos-sdk. 

Actually I realize now that I didn't actually declare it as an official flag anywhere, I just abused the flag mechanism to plumb through an app option. Where do you recommend I declare the `FlagSwingStoreExportDir` flag?

Reply:
I second the confusion, but `FlagSwingStoreExportDir` seems to have strange wiring. At minimum I would look for doc comments.